RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -150,12 +115,55 @@ public class PaywallViewController: UIViewController {
         super.viewDidDisappear(animated)
     }
 
+    /// - Warning: For internal use only
+    @objc(updateWithOffering:)
+    public func update(with offering: Offering) {
+        self.configuration.content = .offering(offering)
+    }
+
+    /// - Warning: For internal use only
+    @objc(updateWithOfferingIdentifier:)
+    public func update(with offeringIdentifier: String) {
```

Comment:
almost wonder whether we should prefix these with `_` or something, or name them something that you're less likely to want to use

Reply:
like, you might be inclined to use the `update(with:)` method if you want to swap the offering under the hood

Reply:
`_` methods are reserved for Apple.

Ultimately there's nothing wrong with others using them though, there's just little reason to need them instead of creating a new `PaywallViewController`, so was just trying to disincentivize potential users of it.